cms/src/main/resources/api-specification/codegen-templates/body_param.mustache

```diff
@@ -1,43 +1,42 @@
 
     {{#mediaTypes}}
 
-        <div class="body__contenttype">
-            <span class="body__contenttype__label">Content type: </span>
-            <span class="body__contenttype__value">{{name}}</span>
-        </div>
+        <p class="nhsd-t-body">
+            Content type: <strong class="nhsd-!t-font-weight-bold">{{name}}</strong>
+        </p>
 
         {{#example}}
-            <div class="body__example__header">Example</div>
-            <div><pre><code>{{this}}</code></pre></div>
+            <p class="nhsd-!t-font-weight-bold nhsd-t-body">Example</p>
+            {{>body_example bodyExample=this}}
```

Comment:
I'VE BEEN LOOKING FOR THIS SYNTAX! How have I missed it?! :roll_eyes: 

Good find :+1: